packages/ui-scripts/lib/build/index.js

```diff
@@ -23,20 +23,25 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-import { readEnv } from '../utils/readEnv.js'
-import { babel } from './babel.js'
-import { examples } from './examples.js'
-import { webpack } from './webpack.js'
-import { clean } from './clean.js'
-
-readEnv()
 
 if (process.argv.includes('--examples')) {
-  examples()
+  // eslint-disable-next-line no-console
+  console.log(
+    'This command is deprecated! Please `use ui-scripts examples` instead'
+  )
 } else if (process.argv.includes('--bundle')) {
-  webpack()
+  // eslint-disable-next-line no-console
+  console.log(
+    'This command is deprecated! Please `use ui-scripts bundle` instead'
+  )
 } else if (process.argv.includes('--clean')) {
-  clean()
+  // eslint-disable-next-line no-console
+  console.log(
+    'This command is deprecated! Please `use ui-scripts clean` instead'
+  )
 } else {
-  babel()
+  // eslint-disable-next-line no-console
+  console.log(
+    'This command is deprecated! Please `use ui-scripts build` instead'
+  )
```

Comment:
Luckily we dont need this part, we've communicated a while ago that auxiliary scripts like these are for internal use only, users should not use them and can change without notice.